package.json

```diff
@@ -30,10 +30,16 @@
     "docs": "pushstate-server docs"
   },
   "lint-staged": {
-    "src/**/*.*.js": [
+    "src/**/*.js": [
       "prettier --no-semi --single-quote --trailing-comma all --write",
       "eslint src --fix",
       "build:docs",
+      "git add docs",
+      "git add"
+    ],
+    "src/**/*.*.js": [
+      "prettier --no-semi --single-quote --trailing-comma all --write",
+      "eslint src --fix",
```

Comment:
@luftywiranda13 Let me double check before I merge this. I was under the same impression, but I think it wasn't respecting that.